testing/file_test/run_test.cpp

```diff
@@ -122,7 +122,7 @@ auto RunTestFile(const FileTestBase& test_base, bool dump_output,
       input_stream = tmpfile();
       fwrite(split->content.c_str(), sizeof(char), split->content.size(),
              input_stream);
-      rewind(input_stream);
+      fseek(input_stream, 0, SEEK_SET);
```

Comment:
This isn't doing anything for us if we don't check the return value though

Reply:
Added `CARBON_CHECK()` since we don't expect this to fail.